opus/application/static_media/js/cart.js

```diff
@@ -112,6 +146,79 @@ var o_cart = {
         });
     },
 
+    updateDownloadFileInfo: function() {
+        /**
+         * Update file info after selecting/deselecting download options.
+         */
+        o_cart.showDownloadSizeSpinner();
+        let add_to_url = o_cart.getDownloadFiltersChecked();
+        o_cart.lastRequestNo++;
+        let url = "/opus/__cart/status.json?reqno=" + o_cart.lastRequestNo + "&" + add_to_url + "&download=1";
+        $.getJSON(url, function(info) {
+            if (info.reqno < o_cart.lastRequestNo) {
+                return;
+            }
+            o_cart.hideDownloadSizeSpinner(info.total_download_size_pretty);
+        });
+    },
+
+    updateCheckboxes: function(target, checkboxesOptions) {
+        /**
+         * Update checkboxes and disable/enable select all & deselect all
+         * buttons correspondingly.
+         */
+        if (($(target).hasClass("op-cart-select-btn") ||
+             $(target).hasClass("op-cart-select-all-btn"))) {
+            for (let checkbox of checkboxesOptions) {
```

Comment:
Since "checkboxesOptions" is the result of a jQuery find(), can't you just do `checkboxesOptions.prop("checked", true);` and it will automatically loop through them all? If not, change the "let" to "const".